src/main/webapp/js/common.js

```diff
@@ -457,7 +457,7 @@ function getPointValue(s, ditchZero) {
     
     if (s0.indexOf('/') !== -1) {
         if (s0.indexOf('S') !== -1) {
-            return 999; // Case N/S
+            return 999; // Case N/S (feedback contribution not sure)
```

Comment:
Yes, that's true. But `N/S` and `N/A` will only appear together with `E +(-)X%` together (0 <= X <= 100).